### PR TITLE
[release/5.0] Fixup 5.0.0 Package shas to be 5.0 RTM

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -43,247 +43,247 @@
...
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>bfc49945c0bedeffe01bb5d6f3c217dad207d0d8</Sha>
+      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
     </Dependency>
     <Dependency Name="Microsoft.Extensions.FileSystemGlobbing" Version="5.0.0">
       <Uri>https://github.com/dotnet/runtime</Uri>
-      <Sha>bfc49945c0bedeffe01bb5d6f3c217dad207d0d8</Sha>
+      <Sha>cf258a14b70ad9069470a108f13765e0e5988f51</Sha>
     </Dependency>
     <Dependency Name="Microsoft.Extensions.HostFactoryResolver.Sources" Version="5.0.0-rtm.20563.12">
```
Comment (dougbu) [inline eng/Version.Details.xml:116]: Shouldn't this be `5.0.0-rtm.20519.4`❔

**Review by dougbu — APPROVED** [review]
Just one question&hellip;

File: eng/Version.Details.xml (hunk 1 above)
Comment (mmitche) [reply eng/Version.Details.xml:116]: I pinged @Anipik on this. The current builds are still producing this package at 5.0.0.

Comment (dougbu) [reply eng/Version.Details.xml:116]: For this PR, my main question wasn't about 5.0.0 versus 5.0.1. Right now, this PR is using the commit SHA for the 5.0.0 release but the `Version` is from a later build.

**Comment by mmitche** [comment]
@dougbu Any ideas about these failures?

**Comment by dougbu** [comment]
@mmitche, @captainsafia has been looking into Selenium-related problems. #27947 may solve things.

I'm not positive but the `Could not write lines to file` problem could be related; that set of tests also uses Selenium.

**Comment by mmitche** [comment]
Any help would be appreciated?

**Comment by captainsafia** [comment]
@jkotalik or @mkArtakMSFT  should be able to help with the merge here.

**Pull request merged**